_posts/2021-09-30-preparing-1.2.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Preparing for Crystal 1.2
+summary: The first release without 32-bit official support
+thumbnail: +
+author: beta-ziliani
+---
+
+We are starting the freeze period⸺only bug fixes from now until the expected release date of Crystal 1.2 starting on the 13th of October. As usual, we invite everyone to test our latest nightly release to ensure that we haven't included a breaking change by mistake ([docker 🐳](https://hub.docker.com/r/crystallang/crystal/tags?page=1&ordering=last_updated&name=nightly), [OS packages 💻](https://crystal-lang.org/install/)).
+
+As of date, we have [162](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.2.0) PRs merged for this version. Thanks to all of those amazing contributors 🙇‍♂️!
+
+On a bit of a sour note, we are demoting our support to 32-bit architectures, moving it from Tier 1 to Tier 2 (that is, _expected to build_). As a result, we will not be shipping i386 packages this time. The reason is that [the specs are failing](https://github.com/crystal-lang/crystal/pull/11096/checks?check_run_id=3331817376), likely due to the amount of memory they consume. Yet, instead of pursuing a solution to the problem, we decided that it was time to continue the trend towards deprecating this architecture, focusing on bringing native support to more popular platforms like ARM64, M1, and Windows. We note this shouldn't preclude anyone from building it from source, or from getting the packages from the distributions that officially support Crystal.
```

Comment:
```suggestion
On a bit of a sour note, we are demoting support for 32-bit x86 architectures, moving it from Tier 1 to Tier 2 (that is, _expected to build_). As a result, we will not be shipping i386 packages this time. The reason is that [the specs are failing](https://github.com/crystal-lang/crystal/pull/11096/checks?check_run_id=3331817376), likely due to the amount of memory they consume. Yet, instead of pursuing a solution to the problem, we decided that it was time to continue the trend towards deprecating this architecture, focusing on bringing native support to more popular platforms like ARM64, M1, and Windows. We note this shouldn't preclude anyone from building it from source, or from getting the packages from the distributions that officially support Crystal.
```